### ISSUE TITLE
Add Setup

### ISSUE THREAD

**Issue by g1ronn1mo** [ExEr7um/vue3-vscode-snippets#53]
There are a lot of snippets out there, but it would be great to have one where you can switch on, off some features. For example the store snippet could be either for vuex or pinia. I use pinia, there is no problem for me, but I think a lot of people are bound to vuex due to company restrictions. 

Best Regards

**Comment by ExEr7um** [comment]
Yeah, I think it's a great idea, I had something similar in mind for a big release. Something like settings, so there will be only `vbase` script that you can configure to your liking.

I will try to do it in future, but PRs are always welcome.
